internal/cli/users.go

```diff
@@ -22,8 +66,341 @@ func usersCmd(cli *cli) *cobra.Command {
+
+			return nil
+		},
+	}
+	userName.RegisterString(cmd, &inputs.Name, "")
+	userConnection.RegisterString(cmd, &inputs.Connection, "")
+	userPassword.RegisterString(cmd, &inputs.Password, "")
+	userEmail.RegisterString(cmd, &inputs.Email, "")
+	userUsername.RegisterString(cmd, &inputs.Username, "")
+
+	return cmd
+}
+
+func showUserCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		ID string
+	}
+
+	cmd := &cobra.Command{
+		Use:   "show",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "Show an existing user.",
+		Long:  "Show an existing user.",
+		Example: `auth0 users show 
+auth0 users show <id>`,
+		PreRun: func(cmd *cobra.Command, args []string) {
+			prepareInteractivity(cmd)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				err := userID.Pick(cmd, &inputs.ID, cli.userPickerOptions)
+				if err != nil {
+					return err
+				}
+			} else {
+				inputs.ID = args[0]
+			}
+
+			a := &management.User{ID: &inputs.ID}
+			if err := ansi.Waiting(func() error {
+				var err error
+				a, err = cli.api.User.Read(inputs.ID)
+				return err
+			}); err != nil {
+				return fmt.Errorf("Unable to load users. The Id %v specified doesn't exist", inputs.ID)
+			}
+
+			cli.renderer.UserShow(a)
+			return nil
+		},
+	}
+
+	return cmd
+}
+
+func deleteUserCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		ID string
+	}
+
+	cmd := &cobra.Command{
+		Use:   "delete",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "Delete a User",
+		Long:  "Delete a User.",
+		Example: `auth0 users delete 
+auth0 users delete <user_id>`,
+		PreRun: func(cmd *cobra.Command, args []string) {
+			prepareInteractivity(cmd)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				err := userID.Pick(cmd, &inputs.ID, cli.userPickerOptions)
+				if err != nil {
+					return err
+				}
+			} else {
+				inputs.ID = args[0]
+			}
+
+			if !cli.force && canPrompt(cmd) {
+				if confirmed := prompt.Confirm("Are you sure you want to proceed?"); !confirmed {
+					return nil
+				}
+			}
+
+			return ansi.Spinner("Deleting User", func() error {
+				_, err := cli.api.User.Read(inputs.ID)
+
+				if err != nil {
+					return fmt.Errorf("Unable to delete User. The specified Id: %v doesn't exist", inputs.ID)
+				}
+
+				return cli.api.User.Delete(inputs.ID)
+			})
+		},
+	}
+
+	return cmd
+}
+
+func updateUserCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		ID 			string
+		Email		string
+		Password	string
+		Name		string
+		Connection	string
+	}
+
+	cmd := &cobra.Command{
+		Use:   "update",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "Update a User",
+		Long:  "Update a User.",
+		Example: `auth0 users update 
+auth0 users update <id> 
+auth0 users update <id> --name John Doe
+auth0 users update -n John Doe --email john.doe@gmail.com`,
+		PreRun: func(cmd *cobra.Command, args []string) {
+			prepareInteractivity(cmd)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
```suggestion
auth0 users update -n John Doe --email john.doe@gmail.com`,
		RunE: func(cmd *cobra.Command, args []string) error {
```
`prepareInteractivity` was moved to the root command last Friday.